fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -2216,4 +2227,5 @@ nonReserved
     | WARNINGS | WEEK | WHITELIST | WORK | WRITE  | WAREHOUSE | WAREHOUSES
     | YEAR
     | DOTDOTDOT
+    | LOCK | UNLOCK | TABLES
```

Comment:
Is it correct to make TABLES as nonReserved?

Reply:
`TABLES` has been defined as a nonReserved token already, here this is a duplicated setting.

Reply:
Removed now.